packages/core-products/db/products/helpers.js

```diff
@@ -60,12 +60,38 @@ const getPriceRange = (prices) => {
 
   return {
     minPrice: {
+      _id: crypto
+        .createHash('sha256')
+        .update(
+          [
+            this._id,
+            Math.random(),
```

Comment:
If you use Math.random here, there is no need for any hash creation. The hashing is thought to make the _id stay the same when the price does not change

Reply:
@pozylon id for min and max price will be the same if i don't find a differentiator. that's why i used it

Reply:
Then use "min" and "max" please. Math.random should never be part of an _id hash